djangocms_googlemap/models.py

```diff
@@ -245,6 +245,8 @@ def get_short_description(self):
             display += '{0}, '.format(self.address)
         if self.lat and self.lng:
             display += '{0} / {1} '.format(self.lat, self.lng)
+        # remove trailing comma if lat or lng are not provided
+        display = display.rstrip(', ')
```

Comment:
this would strip the comma regardless of lat and lng, also why add the comma in the first place if no lat/lon?